README.md

```diff
@@ -1,10 +1,11 @@
 # Gnosis Safe Apps 
 
-[![Logo](https://raw.githubusercontent.com/gnosis/safe-react-apps/main/assets/logo.png)](https://gnosis.pm/)
+[![Logo](https://raw.githubusercontent.com/gnosis/safe-react-apps/main/assets/logo.png)](https://gnosis-safe.io/)
 
-[![Build Status](https://travis-ci.com/gnosis/safe-react-apps.svg?branch=main)](https://travis-ci.com/gnosis/safe-react-apps)
+![license](https://img.shields.io/github/license/gnosis/safe-react-apps)
+![build](https://img.shields.io/github/workflow/status/gnosis/safe-react-apps/Deploy%20safe%20apps/main)
 
```

Comment:
What is your opinion about to add an extra badges for tests:

```
![tests](https://img.shields.io/github/workflow/status/gnosis/safe-react-apps/Test/main?label=tests)
```

And In the future we could add this repo to coveralls (maybe when we finish the new designs in the Tx builder) and show the coverage badge :D

Reply:
Do we have tests?

Reply:
Yes, at least Drain Safe and TX-Builder have some tests

Reply:
and we have tests in the Walletconnect Safe App! :D 
https://github.com/gnosis/safe-react-apps/blob/development/apps/wallet-connect/src/App.test.tsx